install/0000_30_machine-api-operator_11_deployment.yaml

```diff
@@ -64,8 +64,8 @@ spec:
           value: "8080"
         resources:
           requests:
-            cpu: 10m
-            memory: 50Mi
+            cpu: 50m
+            memory: 100Mi
```

Comment:
Why are these bumped suddenly? This seems quite a drastic jump

Reply:
This change allowed this PR to pass the CI. Still not entirely sure why,